File: 1491-average-salary-excluding-the-minimum-and-maximum-salary/1491-average-salary-excluding-the-minimum-and-maximum-salary.py
class Solution:
    def average(self, salary: List[int]) -> float:
        min_sal = 99999999
        max_sal = -99999999
        total = 0
        
        for i in salary:
            if i < min_sal:
                min_sal = i
            if i > max_sal:
                max_sal = i
                
            total += i
        
        return (total-min_sal-max_sal)/(len(salary)-2)

File: 1491-average-salary-excluding-the-minimum-and-maximum-salary/1491-average-salary-excluding-the-minimum-and-maximum-salary.go
func average(salary []int) float64 {
    max_sal := -999999999
    min_sal := 999999999
    total := 0
    
    for i:=0;i<len(salary);i++ {
        if salary[i] < min_sal{
            min_sal = salary[i]
        }
        
        if salary[i] > max_sal{
            max_sal = salary[i]
        }
        
        total += salary[i]
    }
    
    return float64(total - min_sal - max_sal) / float64(len(salary)-2)
    
}